app/assets/javascripts/pages/scenario-form-page.js.coffee

```diff
@@ -0,0 +1,13 @@
+class @ScenarioFormPage
+  constructor:() ->
+    @format
+    @enabledSelect2
+
+  format: (icon) ->
+    originalOption = icon.element
+    '<i class="fa ' + $(originalOption).data('icon') + '"></i> ' + icon.text
+
+  enabledSelect2: () ->
+    $('.select2-fountawesome-icon').select2
+      width: '100%'
+      formatResult: format
```

Comment:
Except for the two minor issue the class definition looks good to me. To initialize it in the scenario edit page `Utils.registerPage` needs to be called:

```
$ ->
  Utils.registerPage(ScenarioFormPage, forPathsMatching: /^scenarios\/\d+\/edit/)
```